Gems/Atom/Feature/Common/Assets/Materials/Types/MaterialFunctions/StandardMaybeClip.azsli

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <Atom/Features/PBR/AlphaUtils.azsli>
+
+void MaybeClip(float alpha, float2 uvs[UvSetCount])
```

Comment:
nit (probably existed before): MaybeClip is not a very good name - can we change it?

Reply:
This is now consolidated with `GetAlpha` to be `GetAlphaAndClip` which I'll push shortly 👍